src/main/java/io/spokestack/spokestack/profile/NoWakeword.java

```diff
@@ -0,0 +1,20 @@
+package io.spokestack.spokestack.profile;
```

Comment:
defining a profile in terms of what it is not is a poor design
every profile is a no-wakeword profile if it doesn't include a wakeword detection component

Reply:
I initially didn't have this one, but I included it because setting an input class is not optional, the TFLiteWakeword class handles the input, and if we have to call `setInputClass` on the builder, the profiles are an incomplete solution. Think I should remove it from the other wakeword profile and just leave it a manual thing?

Reply:
doesn't GoogleSpeechASR also need microphone input?

Reply:
I haven't documented it yet, but the idea is that you'd declare two profiles, one for wakeword, and one for ASR, along with any additional config needed for each. Those are the two main functions handled by the speech pipeline, so it seemed natural to segment them, and have those two profiles replace the cascade of input/stage classes.

Reply:
I don't think I would break up the profiles like that, because that's what you get already from the pipeline builder
I think the profiles should encapsulate all of the default configuration needed for an ensemble of pipeline components, then you can override any properties you want on the builder after attaching the profile